project_template/Makefile.jinja

```diff
@@ -23,7 +23,8 @@ format:  ## Format the code.
 .PHONY: lint
 lint:  ## Run all linters (black/ruff/pylint/mypy).
 	{{ package_manager }} run black --check .
-	{{ package_manager }} run ruff check .
+	{{ package_manager }} run ruff check . || true
+	{{ package_manager }} run pylint {{ module_name }} || true
```

Comment:
The `|| true` at the end of these commands means that the command will always exit with a status of 0 (success), regardless of whether the linter encounters any issues. Since these are used in CI this is not recommended.